examples/security/aws-iam.js

```diff
@@ -0,0 +1,34 @@
+const nodeCloud = require('../../lib/');
+
```

Comment:
formatting is not consistent with other files

Reply:
examples are yet to change after once we have architecture in place.